src/routes/event-match.tsx

```diff
@@ -229,9 +246,28 @@ const EventMatch = ({ eventKey, matchKey }: Props) => {
               />
             </Card>
           )}
-          {/* shows videos if the match has them */}
-          {match.videos && match.videos.length > 0 && (
-            <VideoList videos={match.videos} />
+          {/* shows videos if the match has them, or if offline,
+          a display that only the match details can be displayed */}
+          {isOnline ? (
+            match.videos &&
+            match.videos.length > 0 && <VideoList videos={match.videos} />
+          ) : (
+            <Card
+              class={css`
+                grid-area: analysisTable;
+                justify-self: center;
+                padding: 1rem;
+              `}
+            >
+              <p>
+                <em>
+                  <b>
+                    Analysis table, videos, and reports are only available
+                    online.
+                  </b>
+                </em>
+              </p>
+            </Card>
```

Comment:
@varCepheid Great job with adding this card to inform the user why stuff is missing. I have some feedback on the visual design and text:

- I think it looks better without the italics
- Can we move it to the top above the match info card? Also I think it would look good to have the same width as the other cards.
- Can we add an icon in front of the message? We have an `<Icon>` component to display an icon and you could use [this icon](https://materialdesignicons.com/icon/cloud-off-outline) (or another one)
- What do you think about changing the text like this (the cloud emoji would be the slashed cloud icon):
   :cloud: **No connection** Showing limited information
- Can we use consistent padding all the way around the element? The `<p>` tag is adding extra margin above/below (we don't need a `<p>` tag here).